README.md

```diff
@@ -44,15 +44,19 @@ Start the PCP data logging tools
 
 <table><tbody>
 <tr><th>Type:</th><td><code>scope</code></td><tr><th>Root object:</th><td>PcpInputParams</td></tr>
-<tr><th>Properties</th><td><details><summary>pmlogger_interval (<code>float</code>)</summary>
+<tr><th>Properties</th><td><details><summary>pmlogger_conf (<code>string</code>)</summary>
+                <table><tbody><tr><th>Name:</th><td>pmlogger configuration file</td></tr><tr><th>Description:</th><td>Complete configuration file content for pmlogger as a multi-line string. If no config file is provided, a default one will be generated.</td></tr><tr><th>Required:</th><td>No</td></tr><tr><th>Type:</th><td><code>string</code></td></tbody></table>
```

Comment:
Before each `</tbody>` tag there's a missing `</tr>` tag...although, I'm not sure that anyone cares.

Reply:
https://github.com/arcalot/arcaflow-docsgen/issues/14